languages/javascript/scripts/scaffold/template/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "authors": [
+    {
+      "github_username": "SleeplessByte",
+      "exercism_username": "SleeplessByte"
```

Comment:
I'm not sure we give DJ *all* the credit by default ;) Maybe change these to b empty strings?